### PR TITLE
DOC Document making ModelAdmin easier to work with

### PR DESCRIPTION
**Pull request by GuySartorelli**
Documents the changes from https://github.com/silverstripe/silverstripe-admin/pull/1360

## Parent issue
- https://github.com/silverstripe/silverstripe-admin/issues/1354

### CODE REVIEW

**Comment by GuySartorelli** [comment]
Updated to reflect the changes requested in https://github.com/silverstripe/silverstripe-admin/pull/1360

File: en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md [hunk 1]
```diff
@@ -146,6 +146,26 @@ to be added to your `ModelAdmin`.
...
+    $admin = MyAdmin::singleton();
+    return Controller::join_links(
+        $admin->getLinkForModelTab($admin->getTabForModelClass(static::class)),
+        'cmsPreview',
+        $this->ID
+    );
+}
+```
+
+For earlier versions you'll have you sanitise the class name yourself:
```
Comment (michalkleiner) [inline en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md]:
```suggestion
For earlier versions you'll have to sanitise the class name yourself:
```

File: en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md [hunk 2]
```diff
@@ -161,8 +181,27 @@ public function PreviewLink($action = null)
...
+
+```php
+public function CMSEditLink()
+{
+    $admin = MyAdmin::singleton();
+    return $admin->getEditLinkForDataObject($this);
+}
+```
+
+For earlier versions you'll have you create the link yourself:
```
Comment (michalkleiner) [inline en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md]:
```suggestion
For earlier versions you'll have to create the link yourself:
```

File: en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md [hunk 3]
```diff
@@ -146,6 +146,26 @@ to be added to your `ModelAdmin`.
...
+
+```php
+public function PreviewLink($action = null)
+{
+    if (!$this->isInDB()) {
+        return null;
+    }
+    $admin = MyAdmin::singleton();
+    return Controller::join_links(
+        $admin->getLinkForModelTab($admin->getTabForModelClass(static::class)),
```
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md]:
```suggestion
        $admin->getLinkForModelClass(static::class),
```
I need to update this.

Comment (ChloeSartorelli) [reply en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md]: done

File: en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md (hunk 2 above)
Comment (ChloeSartorelli) [reply en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md]: done

File: en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md (hunk 1 above)
Comment (ChloeSartorelli) [reply en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md]: done

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**